### PR TITLE
Bug 1311997 - Include selection of mail app provider in core ping

### PR DESCRIPTION
**Pull request by bkmunar**

### CODE REVIEW

**Comment by farhanpatel** [comment]
### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
MailProviders.swift | 45 | Colons should be next to the identifier when specifying a type. 
MailProviders.swift | 61 | Colons should be next to the identifier when specifying a type. 
MailProviders.swift | 78 | Colons should be next to the identifier when specifying a type. 
MailProviders.swift | 93 | Colons should be next to the identifier when specifying a type. 
MailProviders.swift | 100 | Colons should be next to the identifier when specifying a type. 
#### Errors

File | Line | Reason |
| --- | ----- | ----- |
ActionViewController.swift | 50 | Lines should not have trailing semicolons. 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: Telemetry/CorePing.swift [hunk 1]
```diff
@@ -93,6 +93,10 @@ class CorePing: TelemetryPing {
             SearchTelemetry.resetCount(profile.prefs)
         }
 
+        if let chosenEmailClient = self.prefs.stringForKey(PrefsKeys.KeyMailToOption) {
+            out["mailClient"] = chosenEmailClient
```
Comment (st3fan) [inline Telemetry/CorePing.swift]: Can you change this to `defaultMailClient` ? This is what we already submitted to the Telemetry people to include in the dashboarding.

Comment (bkmunar) [inline Telemetry/CorePing.swift]: got it

File: Client/Frontend/Browser/OpenWithSettingsViewController.swift [hunk 2]
```diff
@@ -66,17 +66,17 @@ class OpenWithSettingsViewController: UITableViewController {
...
                     previousChoiceAvailable = enabled
                 }
             })
         }
         if !previousChoiceAvailable {
-            self.prefs.setString(mailProviderSource[0].scheme, forKey: "MailToOption")
+            self.prefs.setString(mailProviderSource[0].scheme, forKey: PrefsKeys.KeyMailToOption)
         }
-        self.currentChoice = self.prefs.stringForKey("MailToOption")!
+        self.currentChoice = self.prefs.stringForKey(PrefsKeys.KeyMailToOption)!
```
Comment (st3fan) [inline Client/Frontend/Browser/OpenWithSettingsViewController.swift]: What happens if this pref is not set? Does that mean we crash? Is there a possibility that it is not set? If so, then this should be in a `if let`.

**Pull request merged**